lib/stalePullRequest.js

```diff
@@ -67,11 +67,17 @@ stalePullRequest._processRepository = function (repositoryName, repositorySettin
 
 stalePullRequest._processPullRequest = function (pullRequest, repositorySettings) {
     var commentsUrl = pullRequest.comments_url + '?sort=updated';
+    var commitsUrl = pullRequest.commits_url;
+    var daysSinceLastCommit;
 
     function processComments(commentsJsonResponse) {
         var lastComment = commentsJsonResponse[commentsJsonResponse.length - 1];
+        var pullRequestDate = Cesium.defined(lastComment) ? lastComment.updated_at : pullRequest.updated_at;
         var foundStop = stalePullRequest.foundStopComment(commentsJsonResponse);
-        if (!foundStop && stalePullRequest.daysSince(new Date(lastComment.updated_at)) >= repositorySettings.maxDaysSinceUpdate) {
+        var daysSinceComment = stalePullRequest.daysSince(new Date(pullRequestDate));
+        var daysSinceUpdate = Math.min(daysSinceLastCommit, daysSinceComment);
```

Comment:
The scoping of this function and `daysSinceLastCommit` is a bit confusing.

I think passing in both `commentsJsonResponse` and `commitsJsonResponse` and calling this function `processUpdates` would help the organization here.